scripts/build_scripts/config_reader.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2022 Google LLC
```

Comment:
Nearly a direct copy of the Firebase C++ SDK version, but the comments which contain the  example have been updated with the test app bundle ids / naming convention.